File: universal-utils/src/main/scala/io/github/ppdzm/utils/universal/insurance/FemalePensionExperienceLiveTable2000_2003.scala
package io.github.ppdzm.utils.universal.insurance

/**
 * Created by Stuart Alex on 2021/2/23.
 */
class FemalePensionExperienceLiveTable2000_2003(i: Double) extends ExperienceLiveTable {
    override protected val v: Double = 1 / (1 + i)
    override protected val rows: Map[Int, TableRow] = ExperienceLiveTable.fromDeadList(deadList)
    //TODO
    private val deadList: List[Double] = List()
}

File: universal-utils/src/main/java/io/github/ppdzm/utils/universal/insurance/v2000_2003/FemalePensionExperienceLiveTable2000_2003.java
package io.github.ppdzm.utils.universal.insurance.v2000_2003;

import io.github.ppdzm.utils.universal.insurance.AbstractExperienceLiveTable;

import java.util.List;

/**
 * 女性养老金经验生命表（2000-20032版）
 */
public class FemalePensionExperienceLiveTable2000_2003 extends AbstractExperienceLiveTable {

    public FemalePensionExperienceLiveTable2000_2003(double i) {
        super(i);
    }

    @Override
    public List<Double> deadList() {
        // TODO
        return null;
    }

}
